scripts/bootstrap.sh

```diff
@@ -1,9 +1,30 @@
 #!/bin/bash
 
+# Remember the PWD.
+CWD=$(pwd)
+
 # Clone Notary.
-(cd /tmp; go get github.com/theupdateframework/notary)
+cd $GOPATH/src
+if [ -d "github.com/theupdateframework/notary" ]
+then
+    echo "Notary src already cloned..."
+else
+    mkdir -p github.com/theupdateframework
+    cd github.com/theupdateframework
+    git clone git@github.com:theupdateframework/notary.git
+fi
+
+# Restore PWD.
+cd $CWD
+
+# Build an image containing python-in-toto to verify bundles/images with.
+docker build --rm -t github.com/cnabio/signy/in-toto-container/verification:v1 -f Dockerfiles/in-toto-container.Dockerfile Dockerfiles/
 
 # We will sign and push this to our localhost Notary and Registry.
+echo "Pulling hello-world..."
 docker pull hello-world
 docker tag hello-world localhost:5000/hello-world
+
+echo "Listing all images..."
+docker system prune -f
```

Comment:
I don't think we want this by default in the bootstrap script.

Reply:
Fixed, thanks!